src/gesture.js

```diff
@@ -18,9 +18,11 @@ import {Observable} from './observable';
 import {Pass} from './pass';
 import {devAssert} from './log';
 import {findIndex} from './utils/array';
+import {supportOption} from './utils/event-listener';
 import {toWin} from './types';
 
 const PROP_ = '__AMP_Gestures';
+const passiveSupported = supportOption(self);
```

Comment:
Let's move this into the event helper file and leave it uninitialized. See `optsSupported` in `event-helper-listen.js` as an example.